packages/manager/apps/hub/src/dashboard/dashboard.html

```diff
@@ -59,6 +59,28 @@
                 ></span>
             </a>
         </oui-message>
+        <!-- IAM Banner-->
+        <oui-message
+            data-type="info"
+            data-on-dismiss="$ctrl.onDismiss()"
+            data-dismissable
```

Comment:
**nitpick:** `onDismiss` is not defined in `packages/manager/apps/hub/src/dashboard/dashboard.controller.js`

Reply:
Ok, it was like this on other banners but I fixed it and removed it. Thx